src/data/customer/Background-Job.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview This file contains all background job relatated functions
```

Comment:
So we never have the actual job_id so you could for a specific job? @vinay3206

Reply:
@maartenhemmes in current scenario we don't have a use case where we pass the jobID and get the status.

Currently, when the page is loaded UI makes a call to getTheLatestJob on routing page and check for status.

and if it is in progress then loader will be shown and will be hidden when it will get SNS with success.

When run routing manually then loader is shown will be hidden when SNS is received with success.